src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -89,7 +89,7 @@ codegenModule { name, bindings, imports, foreign: foreign_ } =
 
 definitionIdentifiers :: ChezDefinition -> Array String
 definitionIdentifiers = case _ of
-  DefineValue i _ -> [ i ]
+  Define i _ -> [ i ]
```

Comment:
Reason for this change. If #117 adds macros to `ChezDefinition` via `define-syntax`, then we should make the constructors match the identifiers they represent.